Subject: [PATCH] fixing cmake build error: boost_find_component

---
 cmake_modules/PykepFindBoost.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake_modules/PykepFindBoost.cmake b/cmake_modules/PykepFindBoost.cmake
index 6de9de4..0462a21 100644
--- a/cmake_modules/PykepFindBoost.cmake
+++ b/cmake_modules/PykepFindBoost.cmake
@@ -20,7 +20,7 @@ if(_PYKEP_FIND_BOOST_PYTHON)
     endif()
 endif()
 message(STATUS "Required Boost libraries: ${_PYKEP_REQUIRED_BOOST_LIBS}")
-find_package(Boost ${_PYKEP_BOOST_MINIMUM_VERSION} REQUIRED COMPONENTS "${_PYKEP_REQUIRED_BOOST_LIBS}")
+find_package(Boost ${_PYKEP_BOOST_MINIMUM_VERSION} REQUIRED COMPONENTS ${_PYKEP_REQUIRED_BOOST_LIBS})
 if(NOT Boost_FOUND)
     message(FATAL_ERROR "Not all requested Boost components were found, exiting.")
 endif()
-- 
2.25.1

